tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -74,10 +72,9 @@ def test_send_broadcast_event_does_nothing_if_provider_set_on_service_isnt_enabl
     notify_api,
     sample_service
 ):
-    notify_db.session.add(ServiceBroadcastProviderRestriction(
-        service=sample_service,
-        provider='three'
-    ))
+    settings = ServiceBroadcastSettings(service=sample_service, channel="test", provider="vodafone")
+    settings.provider = "three"
```

Comment:
unless im missing something this can just be

```suggestion
    settings = ServiceBroadcastSettings(service=sample_service, channel="test", provider="three")
```